gog/scope.go

```diff
@@ -259,3 +261,21 @@ func (g *Grapher) assignStructFieldPaths(styp *types.Struct, prefix []string, pk
 		}
 	}
 }
+
+func (g *Grapher) pathEnclosingInterval(start, end token.Pos) ([]ast.Node, bool) {
```

Comment:
I assume this is copy paste from https://github.com/golang/tools/blob/master/go/loader/loader.go#L346 but `g.Files` is the same as all the files looped by `Program`